Scripts/Event/ActiveCollisionsContainerEventProxyEmitter.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace VRTK.Core.Event
+{
+    using UnityEngine.Events;
+    using System;
+    using VRTK.Core.Tracking.Collision.Active;
+
+    /// <summary>
+    /// Emits a UnityEvent with a <see cref="ActiveCollisionsContainer.EventData"/> payload whenever <see cref="Receive"/> is called.
```

Comment:
```suggestion
    /// Emits a UnityEvent with a <see cref="ActiveCollisionsContainer.EventData"/> payload whenever <see cref="SingleEventProxyEmitter{TValue,TEvent}.Receive"/> is called.
```
According to my Visual Studio only this works.